week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,24 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+//Mart.random() returns a random number between 0 (inclusive),  and 1 (exclusive).
+// getNumber() returns a random integer from 0 to 9
 function getNumber() {
     return Math.random() * 10;
 }
 
+
 // Add comments to explain what this function does. You're meant to use Google!
+
+//The concat() method is used to join two or more strings.
```

Comment:
Here must be comment for the function "s".
Please try doing this in VS Code:
1. put /**  --> this must be just 1 line above the function name
2. press enter button
3. see what VS Code will do for you.
Hint: it must be a multi-line comment, parameterised with function's arguments. 
Where you need to shortly specify what the function does at all, and what each of its arguments are for.